ScoutSuite/providers/gcp/facade/base.py

```diff
@@ -8,11 +8,28 @@ def __init__(self, client_name: str, client_version: str):
         self._client_version = client_version
         self._client = None
 
+        # Set logging level to error for GCP services as otherwise generates a lot of warnings
+        logging.getLogger('googleapiclient.discovery_cache').setLevel(logging.ERROR)
+        logging.getLogger().setLevel(logging.ERROR)
+
     def _build_client(self) -> discovery.Resource:
         return discovery.build(self._client_name, self._client_version, 
             cache_discovery=False, cache=MemoryCache())
 
     # Since the HTTP library used by the Google API Client library is not 
     # thread-safe, we need to create a new client for each request.
     def _get_client(self) -> discovery.Resource:
-        return self._build_client()
\ No newline at end of file
+        return self._build_client()
+
+
+class MemoryCache:
+    """
+    Workaround https://github.com/googleapis/google-api-python-client/issues/325#issuecomment-274349841
+    """
+    _CACHE = {}
```

Comment:
Why is this capital? Is it some kind of python convention?

Reply:
I doubt it is, I simply moved the class from another location, it was already like this. I'll rename it!